api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+ 
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+ 
+    <!--
+    	See http://wiki.openmrs.org/display/docs/Module+liquibase+File for 
+    	documentation on this file.
+    
+        See http://www.liquibase.org/manual/home#available_database_refactorings
+        for a list of supported elements and attributes
+    -->
+ 
+ 	<!-- Uncomment this and edit the values below to use this file
+ 	
+	<changeSet id="mymoduleid-2010-05-25-15:50" author="bwolfe">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="mymoduleid_myobject"/></not>
+		</preConditions>
+		<comment>
```

Comment:
Is the commented out code still needed?

Reply:
I've read about it here https://wiki.openmrs.org/display/docs/Module+liquibase+File
and I'm not sure if it's needed or not